ui/apps/platform/src/Containers/NetworkGraph/simulation/NetworkPoliciesYAML.tsx

```diff
@@ -51,7 +51,9 @@ function NetworkPoliciesYAML({
         <CodeEditorControl
             icon={<DownloadIcon />}
             aria-label={labels.downloadYAML}
-            toolTipText={labels.downloadYAML}
+            tooltipProps={{
+                content: labels.downloadYAML,
+            }}
```

Comment:
Would you happen to know if the tooltip was working before? I'll let you decide if adding a test to check the tooltip is worth it.